website/docs/reference/resource-configs/delimiter.md

```diff
@@ -0,0 +1,76 @@
+---
+resource_types: [seeds]
+datatype: <string>
+default_value: ","
+---
+
+## Definition
+
+An optional seed configuration used to separate values in a [seed](/docs/build/seeds) with a string you provide. The delimiter defaults to comma.
+
+
```

Comment:
recommend moving the recommended configs section higher up

```suggestion

* The delimiter defaults to a comma.
* Explicitly set this value if using seed files.
* The `delimiter` configuration should contain a value and not be empty.
```

Reply:
You only need to explicitly set this value if you want your seed files to use a different delimiter (the comma default is the current behavior, and should work for most folks!)

Reply:
gotcha! thanks for clarifying - so maybe:

>Explicitly set this value if using seed files to use a different delimiter (default is comma).